src/OnePlusBot/Modules/SteamProfile.cs

```diff
@@ -31,8 +32,7 @@ public async Task SteampAsync([Remainder] string user)
             {
                 if (stream == null)
                 {
-                    await ReplyAsync("Could not establish connession steam API");
-                    return;
+                    return CustomResult.FromError("Could not establish connession steam API");
```

Comment:
Change error message to something like: `Could not establish connection with the steam API`